src/lib/dev-environment/dev-environment-cli.ts

```diff
@@ -830,43 +830,61 @@ export function processVersionOption( value: unknown ): string {
 export function addDevEnvConfigurationOptions( command: Args ): Args {
 	// We leave the third parameter to undefined on some because the defaults are handled in preProcessInstanceData()
 	return command
-		.option( 'wordpress', 'Use a specific WordPress version', undefined, processVersionOption )
-		.option( [ 'u', 'mu-plugins' ], 'Use a specific mu-plugins changeset or local directory' )
+		.option(
+			'wordpress',
+			'Manage the version of WordPress. Accepts a string value for major versions (6.x). Defaults to the most recent version of WordPress.',
+			undefined,
+			processVersionOption
+		)
+		.option(
+			[ 'u', 'mu-plugins' ],
+			'Manage the source for VIP MU plugins. Accepts "demo" (default) for a read-only image of the staging branch, or a path to a built copy of VIP MU plugins on the local machine.'
+		)
 		.option(
 			'app-code',
-			'Use the application code from a local directory or use "demo" for VIP skeleton code'
+			'Manage the source for application code. Accepts "demo" (default) for a read-only image of WordPress skeleton code, or a path to a git cloned application repo on the local machine.'
 		)
 		.option(
 			'phpmyadmin',
-			'Enable PHPMyAdmin component. By default it is disabled',
+			'Manage PHPMyAdmin, disabled by default. Accepts "y" (default value) to enable or "n" to disable. When enabled, refer to the value of PHPMYADMIN URLS in the information output for a local environment for the URL to access phpMyAdmin.',
```

Comment:
```suggestion
			'Manage phpMyAdmin, disabled by default. Accepts "y" (default value) to enable or "n" to disable. When enabled, refer to the value of "PHPMYADMIN URLS" in the information output for a local environment for the URL to access phpMyAdmin.',
```

We should stay consistent using the [correct capitalization of phpMyAdmin](https://www.phpmyadmin.net/) where possible, and quote the unexpected casing to highlight that's the string that a user should look for in the info output.

Reply:
Whoops! That's a leftover artifact that I overlooked. Thanks!